### PR TITLE
fix: use asset_package for all assets

### LINKED ISSUE #4469: [Bridge/Symfony/Bundle] configurable asset package is not used for every asset
**API Platform version(s) affected**: 2.6.4 (for swagger-ui), all for graphiql and graphiql-playground

**Description**: The asset_path introduced in GH-3764 is not used for every asset

**How to reproduce**: Just set a base_path in your assets.yaml like this:

```yaml
framework:
    assets:
        base_path: '/whatever'
```

**Possible Solution**:  use the asset_package for every asset

**Additional Context**:  original issue #3760

I'll provide a PR for this

### PR DESCRIPTION
**Pull request by mkrauser**
| Q             | A
| ------------- | ---
| Branch?       | 2.6
| Tickets       | fixed #4469
| License       | MIT

Use the configurable asset_path for every asset, not just in SwaggerUi, also in Graphiql and GraphQlPlayground

### CODE REVIEW

File: src/GraphQl/Action/GraphQlPlaygroundAction.php [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 declare(strict_types=1);
 
-namespace ApiPlatform\Core\GraphQl\Action;
+namespace ApiPlatform\GraphQl\Action;
```
Comment (alanpoulain) [inline src/GraphQl/Action/GraphQlPlaygroundAction.php:14]: Why are you changing the namespace?

Comment (mkrauser) [reply src/GraphQl/Action/GraphQlPlaygroundAction.php:14]: Oh, sorry. Must have been some hickup in my IDE. I'll reverse that

File: src/GraphQl/Action/GraphQlPlaygroundAction.php [hunk 2]
```diff
@@ -47,6 +49,7 @@ public function __invoke(Request $request): Response
             return new Response($this->twig->render('@ApiPlatform/GraphQlPlayground/index.html.twig', [
                 'title' => $this->title,
                 'graphql_playground_data' => ['entrypoint' => $this->router->generate('api_graphql_entrypoint')],
+                'assetPackages' => $this->assetPackage,
```
Comment (alanpoulain) [inline src/GraphQl/Action/GraphQlPlaygroundAction.php:52]:
```suggestion
                'assetPackage' => $this->assetPackage,
```

**Review by alanpoulain — APPROVED** [review]

**Comment by alanpoulain** [comment]
Thanks @mkrauser.

**Comment by mkrauser** [comment]
> Thanks @mkrauser.

Thank you for your awesome work @alanpoulain!

**Pull request merged**